apps/server/src/modules/user/service/user.service.ts

```diff
@@ -97,4 +102,18 @@ export class UserService {
 		const promise: Promise<void> = this.userRepo.save(saveEntity);
 		return promise;
 	}
+
+	async migrateUser(currentUserId: string, externalId: string, targetSystemId: string): Promise<void> {
```

Comment:
this feels overly specialized for a function in the user service. And it doesnt in any way specify what kind of migration we are talking about. maybe move it into a more specialized, new service (I htink its fine to have that service in the user module, but its not relevant to everyone using the user service)

Reply:
I moved it to the user-migration service now, think it is fine